docker-for-mac/release-notes.md

```diff
@@ -13,8 +13,8 @@ This page contains information about the new features, improvements, known issue
 
 {% include eula.md %}
 
-## Docker Desktop 3.5.1
-2021-06-25
+## Docker Desktop 3.5.2
```

Comment:
Is this replacing 3.5.1 with 3.5.2?

Reply:
Not really, the 3.5.1 release notes are bellow.

Reply:
Ah it’s the diff getting confused actually…